fxapom/pages/page.py

```diff
@@ -87,6 +86,6 @@ def wait_for_element_not_present(self, *locator):
             WebDriverWait(self.selenium, self.timeout).until(lambda s: len(self.find_elements(*locator)) < 1)
             return True
         except TimeoutException:
-            Assert.fail(TimeoutException)
+            raise TimeoutException
```

Comment:
This is not exactly the same. If we want to just raise the TimeoutException then we shouldn't catch it. If we want to fail here with an assertion error then we should raise an AssertionError.